test/collections.js

```diff
@@ -563,6 +563,8 @@
     equal(3, _.max([1, 2, 3, 'test']), 'Finds correct max in array starting with num and containing a NaN');
     equal(3, _.max(['test', 1, 2, 3]), 'Finds correct max in array starting with NaN');
 
+    equal([3, 6], _.map([[1, 2, 3], [4, 5, 6]], _.max), 'Finds correct max in array when mapping through multiple arrays');
```

Comment:
The implementation just needs to be discussed, we'll let you know if any
other changes are necessary
On Feb 9, 2015 10:39 AM, "Anthony Zotti" <notifications@github.com> wrote:

> In test/collections.js
> <https://github.com/jashkenas/underscore/pull/2039#discussion_r24336399>:
>
> > @@ -563,6 +563,8 @@
> >      equal(3, _.max([1, 2, 3, 'test']), 'Finds correct max in array starting with num and containing a NaN');
> >      equal(3, _.max(['test', 1, 2, 3]), 'Finds correct max in array starting with NaN');
> >
> > +    equal([3, 6], _.map([[1, 2, 3], [4, 5, 6]], _.max), 'Finds correct max in array when mapping through multiple arrays');
>
> What course of action do I need to take in order to make this content high
> value enough to be merged?
>
> Maybe I misunderstood something, I am new to contributing to open source;
> Do I need to refactor my code or my commit history to meet Underscore
> standards?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/2039/files#r24336399>.
>